decidim-core/app/cells/decidim/authorization_modal/show.erb

```diff
@@ -0,0 +1,28 @@
+<div data-dialog-container>
+  <%= icon "lock-line" %>
+  <h3 id="dialog-title-authorizationModal" data-dialog-title><%= title %></h3>
```

Comment:
All modals have an `h3` tag, but they're dynamically placed. That is, in some place could be an h2, somewhere else an h3, etc... so how can we handled that?

Easiest/quickest solution would be, to use a simple `div`. The fact of the title id starts with `id="dialog-title....`, that makes the `aria-labelledby` of the trigger button points to this element.

Is it so important to have an heading tag in a modal? Indeed, the modal contents are "aside" of the main html content, i.e. what a modal title says is not relative to the site content, e.g. if you're in a blog post, _important_ would be the blog title, but a _share_ option would not, right?

Reply:
I think it would have to be conditional based on where they are used. Could we pass the H-level to the cells depending on where they are used?

Otherwise, I would go with `<h2>` as default but then again, on some other pages it may be problematic...

Reply:
Ok, then, addressed in https://github.com/decidim/decidim/issues/10118. I'll resolve all this related reviews here.